sleap_io/model/skeleton.py

```
@@ -7,7 +7,7 @@
 
 from __future__ import annotations
 from attrs import define, field
-from typing import Optional, Tuple, Union
+import typing
```

Ruff

`typing` imported but unused

Remove unused import: `typing`

(F401)